tools/hawk/make_hawk_token.py

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/env python3
```

Comment:
don't forget to kill the old version in the root dir

Reply:
heh, I had moved the file, not `git mv`'d it. d'oh.